src/test/java/com/wikia/webdriver/testcases/adstests/TestAmazonAds.java

```diff
@@ -61,10 +35,10 @@ private void testAmazonAd(String wikiName, String path, boolean debugMode) {
     if (debugMode) {
       testedPage = urlBuilder.appendQueryStringToURL(testedPage, "amzn_debug_mode=1");
     }
-    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, testedPage, networkTrafficInterceptor);
+    AdsAmazonObject amazonAds = new AdsAmazonObject(driver, testedPage);
 
     amazonAds.verifyAmazonScriptIncluded();
-    amazonAds.verifyCallToAmazonIssued();
+//    amazonAds.verifyCallToAmazonIssued();
```

Comment:
So please add ```TODO:
```
or 
```FIXME:
```
with a comment, not a commented code, this one is still in repo history we cann get back to this thing any time